### DISCUSSION TITLE
reporting troyan!

### DISCUSSION THREAD

**Discussion by kambancode** [VSCodium/vscodium#1906] [General]
![image](https://github.com/VSCodium/vscodium/assets/112707802/549e9c07-4c2a-4779-9408-32309a7e40fd)
https://www.virustotal.com/gui/home/upload
check yourself.

**Comment by daiyam** [comment]
Next time, please post more details or the link to the report...

[![Screen Shot 2024-05-28 at 13 50 08](https://github.com/VSCodium/vscodium/assets/587742/1e12a836-3c51-462b-8f1f-d6029ddc5c83)](https://www.virustotal.com/gui/file/60e6b211b026cf854f6a7ee3f08388bd46e50274fb74d55c4f27af7cafe2e2bf)

False positive (*W32.AIDetectMalware*)
